crates/hyperqueue/src/common/cli.rs

```diff
@@ -335,6 +335,10 @@ pub struct JobWaitOpts {
     /// Select job(s) to wait for
     #[arg(value_parser = parse_last_all_range)]
     pub selector: IdSelector,
+
+    /// Wait only for task completion and terminate even job is open
+    #[clap(long, action)]
+    pub without_close: bool,
```

Comment:
Not the best name ever, but can't think of a better one right now, execept for `--do-not-wait-for-close` :)

Reply:
This is my fourth iteration of the name and I am not satisfied, but all other options seems worse to me. I want to have default "more waiting" as it harder to produce silent errors.  `--do-not-wait-for-close` or `--wait-for-close=false` seems to me too long. As an alternative I am proposing `--no-wait-for-close`, but still `without-close` as it may be used after "wait" word (hq wait --without-close ...) seems good to me.

Reply:
:laughing: Yeah, naming is hard. Let's keep the current one then.